src/index.html

```diff
@@ -136,6 +136,14 @@
 <section id="main" class="main">
     <div class="main__wrapper">
         <div class="main__content">
+          <div class="subheader">How many wcs in the property?</div>
+          <div class="select-wrapper">
+            <select class="select">
+              <option value="1">One</option>
+              <option value="2">Two</option>
+              <option value="3">Three</option>
+            </select>
+          </div>
             <div class="subheader">How many bedrooms in the property?</div>
```

Comment:
IMHO i just don't like to include titles/headings inside div tags, because titles should be inside heading tags because they are semantically titles.